src/core/lib/gprpp/thd_posix.cc

```diff
@@ -180,27 +152,6 @@ Thread::Thread(const char* thd_name, void (*thd_body)(void* arg), void* arg,
     *success = outcome;
   }
 }
-
-void Thread::Init() {
```

Comment:
These methods should also be removed from `thd.h` and `thd_windows.cc` if they're no longer part of the thread management API.

Reply:
Fixed